tests/unittests/MLTest.cpp

```diff
@@ -1403,6 +1403,79 @@ TEST_P(MLTest, matrixRotationRecognition) {
   EXPECT_LE(errors, 1);
 }
 
+/// Simple test case that learns the embedding table for a
+/// SparseLengthsWeightedSum operator.
+TEST_P(InterpreterAndCPU, learnSparseLengthsWeightedSum) {
+  TrainingConfig TC;
+  TC.learningRate = 0.3;
+  TC.batchSize = 1;
+
+  PlaceholderBindings bindings;
+
+  Module &mod = EE_.getModule();
+  PseudoRNG &PRNG = mod.getPRNG();
+
+  // Create a model consisting of one SparseLengthsWeightedSum operator followed
+  // by a Regression node to get some non-zero gradients.
+  Function *F = mod.createFunction("SparseLengthsWeightedSum");
+  Placeholder *dataP =
+      mod.createPlaceholder(ElemKind::FloatTy, {10}, "dataP", true);
```

Comment:
nit: I know we don't use it super often, but I'd prefer adding `/* isTrainable */ true`, and so on especially since we are actually training here.